amplify/backend/function/AddItem/AddItem-cloudformation-template.json

```diff
@@ -0,0 +1,292 @@
+            },
+        
+    
+        
+    
+        "storageitemsName": {
+            "Type": "String",
+            "Default": "storageitemsName"
+            },
+        
+        "storageitemsArn": {
+            "Type": "String",
+            "Default": "storageitemsArn"
+            },
+        
+        "storageitemsStreamArn": {
+            "Type": "String",
+            "Default": "storageitemsStreamArn"
+            },
+        
+    
+        
+    
+        "storagebatchName": {
+            "Type": "String",
+            "Default": "storagebatchName"
+            },
+        
+        "storagebatchArn": {
+            "Type": "String",
+            "Default": "storagebatchArn"
+            },
+        
+        "storagebatchStreamArn": {
+            "Type": "String",
+            "Default": "storagebatchStreamArn"
+            },
+        
+    
+        
+    
+        "storagetagsName": {
+            "Type": "String",
+            "Default": "storagetagsName"
+            },
+        
+        "storagetagsArn": {
+            "Type": "String",
+            "Default": "storagetagsArn"
+            },
+        
+        "storagetagsStreamArn": {
+            "Type": "String",
+            "Default": "storagetagsStreamArn"
+            },
+        
+    
+        
+    
+        "storagehistoryName": {
+            "Type": "String",
+            "Default": "storagehistoryName"
+            },
+        
+        "storagehistoryArn": {
+            "Type": "String",
+            "Default": "storagehistoryArn"
+            },
+        
+        "storagehistoryStreamArn": {
+            "Type": "String",
+            "Default": "storagehistoryStreamArn"
+            },
+        
+    
+        
+    
+        "storagetransactionsName": {
+            "Type": "String",
+            "Default": "storagetransactionsName"
+            },
+        
+        "storagetransactionsArn": {
+            "Type": "String",
+            "Default": "storagetransactionsArn"
+            },
+        
+        "storagetransactionsStreamArn": {
+            "Type": "String",
+            "Default": "storagetransactionsStreamArn"
+            }
+        
+    
+        
+    
+    },
+    "Conditions": {
+        "ShouldNotCreateEnvResources": {
+            "Fn::Equals": [
+                {
+                    "Ref": "env"
+                },
+                "NONE"
+            ]
+        }
+    },
+    "Resources": {
+        "LambdaFunction": {
+          "Type": "AWS::Lambda::Function",
+          "Metadata": {
+            "aws:asset:path": "./src",
+            "aws:asset:property": "Code"
+          },
+          "Properties": {
+            "Code": {
+                "S3Bucket": {
+                    "Ref": "deploymentBucketName"
+                },
+                "S3Key": {
+                    "Ref": "s3Key"
+                }
+            },
+            "Handler": "./ts-output/handers/api/AddItem.js",
```

Comment:
`.handler` instead of `.js`. Essentially, we want to point to the 'handler' function inside of AddItem